includes/embeds/class-amp-youtube-embed.php

```diff
@@ -64,6 +64,12 @@ public function shortcode( $attr ) {
 	}
 
 	public function oembed( $matches, $attr, $url, $rawattr ) {
+		// Use the standard oembed handler if it's a playlist
+		$parsed_url = wp_parse_url( $url );
+		if ( isset( $parsed_url['path'] ) && false !== strpos( $parsed_url['path'], 'playlist' ) ) {
+			return wp_oembed_get( $url );
```

Comment:
Update: we might be able to skip all of this by just removing `playlist` from [the regex](https://github.com/Automattic/amp-wp/blob/0.3.3/includes/embeds/class-amp-youtube-embed.php#L8) in the Youtube embed handler. That seems to have the same effect, because then the pattern doesn't match and it just gets caught later by the standard oembed. I'm guessing now that this is what you were suggesting above, @mjangda ;)

Thoughts? We could just close this pull request and I could open another one with a one-line change.